app/src/main/kotlin/com/wire/android/ui/common/UserProfileAvatar.kt

```diff
@@ -77,14 +44,19 @@ fun UserProfileAvatar(
             .wrapContentSize()
             .padding(MaterialTheme.wireDimensions.userAvatarClickablePadding)
     ) {
+        val avatarResource = avatarAssetByteArray?.run {
+            BitmapFactory.decodeByteArray(avatarAssetByteArray, 0, avatarAssetByteArray.size)
+        } ?: getDefaultAvatarUri(LocalContext.current)
+
         Image(
-            bitmap = avatarBitmap.asImageBitmap(),
+            painter = rememberAsyncImagePainter(model = avatarResource),
             contentDescription = stringResource(R.string.content_description_user_avatar),
-            contentScale = ContentScale.Crop,
             modifier = Modifier
                 .padding(dimensions().userAvatarStatusBorderSize)
-                .clip(CircleShape)
+                .background(Color.Companion.Black, CircleShape)
```

Comment:
could you at least change it to Color.Black :P you do not need calling Companion there